sass/blocks/intro/_intro.scss

```diff
@@ -44,4 +43,19 @@
     background-size: auto;
     background-position: 50% 100%;
   }
+
+  // Tablet screen
+  @media (min-width: 480px) {
```

Comment:
I think Tablet should start at the size of a design, it's too early it breaks the nav
![image](https://user-images.githubusercontent.com/1570578/36917341-692b1ece-1e4e-11e8-8d0e-802e664e1d00.png)